src/features/dashboard/ui/ResponsesList.tsx

```
@@ -1,121 +1,95 @@
-                  ? { v1: selectedResponse[0].name, v2: selectedResponse[0].email }
-                  : { v1: '전체', v2: '' }
-              }
-              setSelectedField={setSelectedResponse}
-              setCurrentIndex={setCurrentIndex}
-              currentIndex={currentIndex}
-              responsesLength={selectedResponse.length > 0 ? selectedResponse.length : response.length}
-              options={[{ v1: '전체', v2: '' }, ...response.map(res => ({ v1: res.name, v2: res.email }))]}
-            />
-            <SelectedResponseList currentIndex={currentIndex} />
-          </div>
+          <IndividualResponseViewer
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)